src/ResourceManager/Insights/Commands.Insights/MonitorCmdletBase.cs

```diff
@@ -33,6 +33,29 @@ public abstract class MonitorCmdletBase : AzureRMCmdlet
         protected abstract void ProcessRecordInternal();
 
         /// <summary>
+        /// Writes a warning message with the name of the cmdlet, a topic and the message itself
+        /// </summary>
+        /// <param name="topic">The topic, i.e. short description/category, of the message</param>
+        /// <param name="message">The message itself</param>
+        /// <param name="withTimeStamp">true if the message should include a timestamp, false (default) it no timestamp should be included</param>
+        protected void WriteIdentifiedWarning(string topic, string message, bool withTimeStamp = false)
+        {
+            string formattedMessage = Utilities.FormatIdentifiedMessage(
```

Comment:
@gucalder we are going to be creating our own mechanism for custom obsolete messages and attributes soon, similar to what you are doing here, so we recommend just making this simple for the time being since it will change in the near future ☺️